### DISCUSSION TITLE
Release with Linux 6.0.x support - ETA?

### DISCUSSION THREAD

**Discussion by eirnym** [openzfs/zfs#14227] [Q&A] [answered]
Thank you all for your hard work on supporting a driver for this filesystem

Linux 6.0 support was officially set in master https://github.com/openzfs/zfs/commit/871d66dbf2bee4e2aff9346330b3b5add7c44562 (29 days ago) and in 2.1.7 release branch https://github.com/openzfs/zfs/commit/7ce097c8747a403f3ddf8c2f6be4ac486b959d8a (24 days ago). Looks like release process has been forgotten or not clearly communicated. Last few releases I observe similar situation, previously there was no such problems and it looked like that new kernel is out and few days (not months!) later I have zfs update in my distribution (arch linux)

Why this situation is happening? Could you please tell when release is planned?

**Comment by thulle** [comment, marked as answer]
See #14162

**Reply by eirnym** [reply]
Thank you! I miss this issue!
